src/arkouda_server.chpl

```diff
@@ -173,6 +173,19 @@ proc main() {
                 authenticateUser(token);
             }
 
+            if (logging) {
+              try {
+                if (cmd != "array") {
+                  writeln(">>> %s %s".format(cmd, payload.decode(decodePolicy.replace)));
+                } else {
+                  writeln(">>> %s [binary data]".format(cmd));
+                }
+                stdout.flush();
+              } catch {
+                // No action on error
```

Comment:
Is there any action that should be taken if there is an error in the payload.decode invocation? If so, I recommend two catch blocks, one for the type of Error thrown in the payload.decode invocation

Reply:
I thought about that, but if there's a payload decode error, it should be caught by the message function specific to the command, with presumably a more specialized error message, so I figured I should let it fall through. Thoughts?